bricks/template/hooks/.gitignore

```diff
@@ -0,0 +1,2 @@
+.dart_tool
+build
```

Comment:
line break for consistency?

Reply:
This is optional, up to you @manh-t

Reply:
@doannimble  thanks, updated in [de4d201](https://github.com/nimblehq/flutter-templates/pull/203/commits/de4d201fb66a46cf82bfb847c66ddeeefc349f8b)